web/src/main/webapp/WEB-INF/config-csw.xml

```diff
@@ -108,7 +108,7 @@
 
         <parameter name="Language" field="language" type="SupportedISOQueryables">
           <xpath schema="iso19139"
-                 path="gmd:language/gco:CharacterString|gmd:language/gmd:LanguageCode/@codeListValue"/>
+                 path="if (normalize-space(gmd:language/gmd:LanguageCode/@codeListValue) != '') then gmd:language/gmd:LanguageCode/@codeListValue else gmd:language/gco:CharacterString"/>
```

Comment:
@ianwallen I tested the pull request and this change doesn't work, should be the previous code or if case want to check first the LanguageCode switch the order, but I guess that's not relevant as `gmd:language` will have only `gco:CharacterString` or `gmd:LanguageCode`, but not both.

Please update the code and will merge it, thanks.